### PR TITLE
Fix config path for config.toml to run mainnet node

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Fixed config.toml path to run mainnet node.

Files changed: 

- docs/get-started/running-api-node.md
- docs/get-started/running-mainnet-node.md
- docs/stacks-blockchain-api/how-to-guides/how-to-run-stacks-blockchain-api-docker.md

Relates to an issue opened by an external dev: https://github.com/hirosystems/docs/issues/221

### CODE REVIEW

**Review by rafaelcr — APPROVED** [review]
LGTM, I think @CharlieC3 's review would also be very valuable here though 🙏

**Review by CharlieC3 — APPROVED** [review]
Just tested the change locally and it fixed it! Left a couple comments on what may be missing or unused pages.

File: docs/get-started/running-api-node.md [hunk 1]
```diff
@@ -155,8 +155,7 @@ docker ps --filter name=stacks-blockchain-api
 ## Step 4: Running Stacks blockchain
```
[the remark this answers is not in the document]
Comment (LakshmiLavanyaKasturi) [reply docs/get-started/running-api-node.md:155]: Thanks for the find! Missed adding them to the sidebar during our re-org. I will fix them right away.

File: docs/get-started/running-mainnet-node.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ mkdir -p ./stacks-node/{persistent-data/stacks-blockchain/mainnet,config/mainnet
```
[the remark this answers is not in the document]
Comment (LakshmiLavanyaKasturi) [reply docs/get-started/running-mainnet-node.md:61]: Thanks for the find! Missed adding them to the sidebar during our re-org. I will fix them right away.

**Pull request merged**